src/app.rb

```diff
@@ -0,0 +1,85 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './student'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person(name, age, type)
+    if type == 'student'
+      puts 'Has a parent permission? [Yes/No]'
+      parent_permission_q = gets.chomp.downcase
+
+      if parent_permission_q == 'yes'
+        parent_permission = true
+        person = Student.new(name, age, parent_permission: parent_permission)
+      else
+        parent_permission = false
+      end
```

Comment:
- [ ] When creating student person the question about parent´s permission appeared twice if the answer was "Yes", also the confirmation message was repeated twice, please 🙏🏽 correct this issue for a better UX.

![image](https://github.com/zhorabay/School-library/assets/58642949/9ca37862-e9c6-4dc9-8202-5d5024962ef4)